build-tools/packages/bundle-size-tools/package.json

```diff
@@ -10,6 +10,13 @@
 	},
 	"license": "MIT",
 	"author": "Microsoft and contributors",
+	"exports": {
+		".": {
+			"types": "./lib/index.d.ts",
+			"import": "./lib/index.js",
```

Comment:
I don't think these are right. This package doesn't build ESM, and its CJS output is in dist.